src/celpy/evaluation.py

```diff
@@ -51,6 +51,14 @@
 import celpy.celtypes
 from celpy.celparser import tree_dump
 
+_USE_RE2 = False
+try:
+    import re2
+except ImportError:
+    pass
+else:
```

Comment:
This seems a bit convoluted. Can we put the `_USE_RE2 = True` after the `import` and lose this `else:` block?